src/components/screens/InfoCard.js

```diff
@@ -113,15 +124,18 @@ export const InfoCard = ({ addresses, category, contact, contacts, webUrls }) =>
 
     {!!webUrls &&
       webUrls.map((item, index) => {
-        const { url } = item;
+        const { url, description } = item;
 
-        if (!url) return null;
+        if (!url) {
+          return null;
+        }
 
         return (
           <InfoBox key={index}>
             <Icon icon={urlIcon(colors.primary)} style={styles.margin} />
             <TouchableOpacity onPress={() => openLink(url)}>
-              <RegularText link>{url}</RegularText>
+              {!description && <RegularText link>{url}</RegularText>}
+              {!!description && <RegularText link>{description}</RegularText>}
```

Comment:
Well done, we will take this!

But :)
Now there is a duplication of the `<RegularText link>...</RegularText>`. Only the `...` data inside of the RegularText is different. Though, we could think of having only one and making the data conditionally.

This could look something like this:

`<RegularText link>{!!description ? description : url}</RegularText>` what we read: if there is a `description`, use the `description`, otherwise use the `url`.

Another option would be to declare a `const linkText` conditionally before the `return` and use that inside `<RegularText link>{linkText}</RegularText>`, where `linkText` is set to `description` or `url` depending on the data.